README.md

```diff
@@ -1,17 +1,61 @@
 # Northcoders News API
 
+## Project Aims ##
 An example API which accesses application data programmatically. The intention here is to mimic the building of a real world backend service (such as Reddit) which should provide this information to the front end architecture.
 
-Your database will be PSQL, and you will interact with it using node-postgres.
+## Link To Hosted Project ##
+https://smarticles.onrender.com/api/articles/
 
-You will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=, with the relevent database name for that environment. Please ensure that these .env files are .gitignored.
+## Project Setup ##
+
+To replicate this project:
+
+Your database will be PSQL, this will be interacted with using node-postgres.
+
+#### Creating .env files ####
+You will need to create two .env files for your project: .env.test and .env.development. Into each, 
+add PGDATABASE=<>
+with the relevent database name for that environment. __Please ensure that these .env files are .gitignored.__ 
```

Comment:
I would name the relevant db's here, just for ease of use of people using your repo.

This is important because this project will only work with specific names